src/scss/_mixins.scss

```diff
@@ -194,6 +75,36 @@
 	}
 }
 
+/// Set a custom font.
+///
+/// @example This example shows setting a custom font "MySansFont" as the "sans" font. We specify out custom font supports regular or bold variants.
+/// 	@include oTypographySetCustomFont($family-style: 'sans', $family: 'MySansFont, sans', $variants: (
+/// 		(weight: regular, style: normal),
+/// 		(weight: bold, style: normal)
+/// 	));
+///
+/// @param {String} $family-style - One of 'sans', 'serif', or 'display'.
+/// @param {String} $family - The font family to set.
+/// @param {Map} $variants - The variants (weight and style combinations) which are allowed in a nested map.
+@mixin oTypographySetCustomFont($family-style, $family, $variants) {
+	$family-styles: ('sans', 'serif', 'display');
+	@if not index($family-styles, $family-style) {
+		@error 'Could not set font-family "#{$family}" for family style "#{$family-style}", family style must be one of "#{$family-styles}".';
+	}
+	// Register the custom font with o-fonts.
+	@include oFontsDefineCustomFont($family, $variants);
+	// Set the custom font.
+	@if $family-style == 'sans' {
```

Comment:
just to double check—we'd need to call this mixin multiple times if we want to set various different fonts, correct?

Reply:
Yeah 👍